packages/office-addin-debugging/src/start.ts

```diff
@@ -10,6 +11,8 @@ import * as nodeDebugger from "office-addin-node-debugger";
 import { getProcessIdsForPort } from "./port";
 import { startDetachedProcess, startProcess  } from "./process";
 
+export const processIdFile = "devServerProcessId.txt";
```

Comment:
I don't think we should just create the file in the current directory. It could stick around.

We might want a way to have a central place to put the file and have a mapping from the directory where run or from the add-in id to the devServerProcessId. This could be created in a temp folder.

Reply:
I can make this change, but want to make sure this is needed.  I believe we are expecting the users to run npm run start and npm run stop from the same directory.  Is this correct?  Also with this design even if the file is not deleted after closing the process, the file will be overwritten the next time.  User should not get multiple files left around.  If we go with a design of a unique name for the file and it doesn't get deleted because the user manually closes the process, they can end up with multiple files left around. I could design it to delete all the files in the temp folder when starting the process and write a new file, thus making sure we don't have multiple files left around. This would be a little more complex, but doable. Let me know which design you prefer.